app/api/v1/blueprints/parties.py

```diff
@@ -30,10 +30,16 @@ def create_party():
         return jsonify(response), 201
     elif result == 'Not authorised':
```

Comment:
the value of result on line 31 should not be **Not authorised**

Reply:
Rectified